pyat/at/matching/matching.py

```diff
@@ -41,8 +43,8 @@ class Variable(object):
     """
     def __init__(self, setfun: Callable, getfun: Callable,
                  name: str = '',
-                 bounds: Tuple[float, float] = (-np.inf, np.inf),
-                 *args, **kwargs):
+                 bounds: tuple[float, float] = (-np.inf, np.inf),
+                 args: tuple = (), **kwargs):
```

Comment:
Detail: to avoid confusion with the usual `*args` (variable argument list), would this  be better named `fun_args`, or something better?